client/app/bundles/course/assessment/attemptLoader.ts

```diff
@@ -0,0 +1,14 @@
+import { LoaderFunction, redirect } from 'react-router-dom';
+import { getIdFromUnknown } from 'utilities';
+
+import CourseAPI from 'api/course';
+
+const assessmentAttemptLoader: LoaderFunction = async ({ params }) => {
+  const assessmentId = getIdFromUnknown(params?.assessmentId);
+  if (!assessmentId) return redirect('/');
+
+  const { data } = await CourseAPI.assessment.assessments.attempt(assessmentId);
+  return redirect(data.redirectUrl);
```

Comment:
When the backend cant create a submission and returns an error, it will be caught by the error boundary instead of staying on the page, warn the user to try again?